extensions/typescript/src/protocol.d.ts

```diff
@@ -1,2 +1,2 @@
-import * as Proto from '../node_modules/typescript/lib/protocol';
+import * as Proto from '../../node_modules/typescript/lib/protocol';
```

Comment:
Use `import * as Proto from 'typescript/lib/protocol';` instead to avoid the path dependency